collector/README.md

```diff
@@ -167,6 +171,22 @@ something like this:
 where `$RUST_ORIGINAL` and `$RUST_MODIFIED` are paths (relative or absolute) to
 the relevant rustc executables.
 
+#### Runtime benchmarks
+There is also a runtime benchmark suite, which measures the performance of Rust programs compiled
+by a selected version of rustc. You can run it using the following command:
+```bash
+./target/release/collector bench_runtime_local <RUSTC>
+```
+
+### Benchmarking options
+
+The following options alter the behaviour of the `bench_runtime_local` subcommand.
+- `--no-isolate`: you can use this flag to make repeated local benchmarks faster. It will cause the
+  `collector` to reuse compiled artifacts of the runtime benchmark groups.
```

Comment:
Are there any potential negative effects from this flag? E.g. worse reproducibility?